.changeset/rude-drinks-relate.md

```diff
@@ -0,0 +1,8 @@
+---
+"@atproto/dev-env": patch
+"@atproto/ozone": patch
+"@atproto/bsky": patch
+"@atproto/pds": patch
+---
+
+Updated to use refactored version of `@atproto/api`.
```

Comment:
Same here, if it's truly just for a version update then changesets will automatically detect that and add a note to the changelog automatically.

Reply:
Certainly wont do any harm though, so not terribly worried about it!